.github/workflows/terraform.yaml

```diff
@@ -17,6 +17,11 @@ on:
         type: string
         default: ""
         required: false
+      checkov_output_format:
+        description: Checkov output format
+        type: string
+        default: "sarif"
+        required: false
```

Comment:
@Dineshk77 is SARIF readable by humans? We don't have SARIF upload enabled for all repos (cost reason).